plasma-roundup/draft-plasma-article.md

```diff
@@ -0,0 +1,76 @@
+
+This overview is not meant to be exhaustive. Instead, by highlighting a few implementations, it aims to be indicative of the progress that Plasma has made over the past year.
+
+Although none of the Plasma models reviewed here are on mainnet or production ready, they demonstrate that this technology is not just theory. Taken together, the many implementations suggest that Plasma is moving rapidly toward realizing the scaling potential that its creators and implementers envision.     
+
+Among the early Plasma implementations was [FourthState Labs](https://github.com/FourthState/plasma-mvp-rootchain), whose design included a rootchain contract according to the Plasma MVP. This rootchain, which has been incorporated into other implementations, is pretty simple to test. To test, I used this environment:
+
+* Linux (Debian Stretch)
+* Truffle (v5.0.2)
+* Ganache CLI (v6.2.5)
+* NPM 6.7.0
+
+Follow these steps to test:
+
+0. `$ git clone https://github.com/fourthstate/plasma-mvp-rootchain`
+1. `$ cd plasma-mvp-rootchain`
+2. `$ npm install`
+3. `$ npm install -g truffle ganache-cli` // if not installed already
+4. `$ ganache-cli` // run as a background process
+5. `$ npm test`
+
+![FourthState tests](/images-for-article/Fourth-Estate/fourth-estate.png)
+
+Other notable early MVP's include [OmiseGO's research implementation](https://github.com/omisego/plasma-mvp), which has a root chain, child chain and a client to interact with the Plasma chain. Moving beyond research, [Kyokan](https://github.com/kyokan/plasma) introduced their Golang implementation [extending the original MVP specification](https://kauri.io/article/7f9e1c04f3964016806becc33003bdf3/v4/minimum-viable-plasma-the-kyokan-implementation). Kyokan uses the FourthState rootchain contract reviewed above. The architecture includes root nodes to process transactions and package them into blocks, broadcasts blocks to validator nodes, processes exits and more.
+
+In Kyokan, validator nodes check the validity of blocks and exit if bad behavior is detected. The Plasma contract lives on the Ethereum root chain and supports deposits, block submissions, exits and challenges. Kyokan, which aims to provide a pluggable architecture, has deployed their [Plasma Block Explorer](https://explorer.kyokan.io/) on the Rinkeby test net:
+
+![Kyokan Block Explorer](/images-for-article/Kyokan/kyokan-block-explorer.png)
+
+Closing 2018, the [Plasma Group](https://plasma.group/) [announced the release of their implementation](https://medium.com/plasma-group/plasma-spec-9d98d0f2fccf) aimed at the greater Ethereum community. It includes a Plasma chain operator, a client and command line wallet, support for ERC20 tokens, a block explorer, transaction load testing and more.
```

Comment:
```suggestion
Closing 2018, the [Plasma Group](https://plasma.group/) [announced the release of their implementation](https://medium.com/plasma-group/plasma-spec-9d98d0f2fccf) aimed at the greater Ethereum community. It includes a Plasma chain operator, a client and command line wallet, support for ERC20 tokens, a block explorer, transaction load testing and more. While their implementation includes properties such as scalable light client proofs and the possibility for interchain atomic swaps, the group has moved quickly to offer a [general purpose plasma design](https://medium.com/plasma-group/towards-a-general-purpose-plasma-f1cc4d49c1f4). This general purpose design aims to overcome constraints in old design patterns which were not upgradeable and not generalizable. The purpose is to create a plasma environment that would allow the development of applications on a Plasma chain in the same way that dapps are built on the Ethereum blockchain. Hence the Plasma Group has recently offered plasma apps, also known as ‘plapps’. Plapps are a special type of smart contract called a ‘[predicate contract](https://github.com/plasma-group/plasma-predicates)’. Plapps are deployed to Ethereum, where users can interact with the contract and the plasma chain carries out the computation, which is less expensive than if it were to occur on the Ethereum main chain. 
```